content/en/docs/tasks/security/authentication/authn-policy/index.md

```diff
@@ -625,174 +471,72 @@ $ for i in `seq 1 10`; do curl --header "Authorization: Bearer $TOKEN" $INGRESS_
 You can also add a JWT policy to an ingress gateway (e.g., service `istio-ingressgateway.istio-system.svc.cluster.local`).
 This is often used to define a JWT policy for all services bound to the gateway, instead of for individual services.
 
-### End-user authentication with per-path requirements
-
-End-user authentication can be enabled or disabled based on request path. This is useful if you want to
-disable authentication for some paths, for example, the path used for health check or status report.
-You can also specify different JWT requirements on different paths.
+### Enforce valid token must be presented.
 
-{{< warning >}}
-The end-user authentication with per-path requirements is an experimental feature in Istio 1.1 and
-is **NOT** recommended for production use.
-{{< /warning >}}
-
-#### Disable End-user authentication for specific paths
-
-Modify the `jwt-example` policy to disable End-user authentication for path `/user-agent`:
+To reject request without valid token, you will need to add an authorization rule to `deny` request without `requestPrincipal` as below. Request principal is available only when a valid JWT token is provided, so the rule is equivalent to deny requests without valid token.
 
 {{< text bash >}}
-$ cat <<EOF | kubectl apply -n foo -f -
-apiVersion: "authentication.istio.io/v1alpha1"
-kind: "Policy"
+kubectl apply -f - <<EOF
+apiVersion: "security.istio.io/v1beta1"
+kind: "AuthorizationPolicy"
 metadata:
-  name: "jwt-example"
+  name: "frontend-ingress"
+  namespace: istio-system
+  labels:
+    demo.istio.io: security
```

Comment:
why is this label?

Reply:
Want to make the clean up command simpler. Remove it for now